local/protocol/local.proto

```diff
@@ -22,4 +25,37 @@ message EchoRequest {
 
 message EchoReply {
   string pong = 1;
+}
+
+enum ProcessState {
+  UNKNOWN = 0;
+  PENDING = 1;
+  RUNNING = 2;
+
+  COMPLETED = 3;
+  FAILED = 4;
+}
+
+message Duration {
+  int64 seconds = 1;
```

Comment:
Ummm, it's what Google does? https://github.com/google/protobuf/blob/master/src/google/protobuf/duration.proto

But I think you're right, it's silly. I'll remove it.